### ISSUE TITLE
Use Data

### ISSUE THREAD

**Issue by zekhoi** [lonqie/SchaleDB#23]
May i use your data in this repo and store it in my database?

**Comment by lonqie** [comment]
Yes it is ok, but the format is subject to changes without any notice so please keep that in mind.
